lib/sensorLayouts/index.js

```diff
@@ -7,6 +7,14 @@ const senseboxhome = require('./sensebox.home'),
   luftdaten_sds011_dht22 = require('./luftdaten/sds011.dht22'),
   luftdaten_sds011_bmp180 = require('./luftdaten/sds011.bmp180'),
   luftdaten_sds011_bme280 = require('./luftdaten/sds011.bme280'),
+  luftdaten_pms1003 = require('./luftdaten/pms1003'),
+  luftdaten_pms1003_bme280 = require('./luftdaten/pms1003.bme280'),
+  luftdaten_pms3003 = require('./luftdaten/pms3003'),
+  luftdaten_pms3003_bme280 = require('./luftdaten/pms3003.bme280'),
+  luftdaten_pms5003 = require('./luftdaten/pms5003'),
+  luftdaten_pms5003_bme280 = require('./luftdaten/pms5003.bme280'),
+  luftdaten_pms7003 = require('./luftdaten/pms7003'),
+  luftdaten_pms8003_bme280 = require('./luftdaten/pms7003.bme280'),
```

Comment:
this should be `luftdaten_pms7003_bme280` instead of `luftdaten_pms8003_bme280`

Reply:
should be fixed now